src/node_platform.cc

```diff
@@ -244,14 +244,22 @@ void PerIsolatePlatformData::PostIdleTask(std::unique_ptr<v8::IdleTask> task) {
 }
 
 void PerIsolatePlatformData::PostTask(std::unique_ptr<Task> task) {
-  CHECK_NOT_NULL(flush_tasks_);
+  if (flush_tasks_ == nullptr) {
+    // V8 may post tasks during Isolate disposal. In that case, the only
```

Comment:
I wonder if a Debug print here would be helpful in the future..

Reply:
What would that debug print print?

Reply:
"Task discarded since it's posted during Isolate disposal"? Just in case this race triggers something surprising in the future and perhaps we could tell the user to try turning on the debug variable if it's too difficult to provide a reproducible test case for us.

Reply:
If you’re suggesting to put printing debugging information for the platform impl behind a CLI flag, that sounds okay to me

Reply:
I'm not convinced of the usefulness of such a debug print. The definition of a foreground task in V8 is more or less that it's a task that uses the isolate. If it didn't use the isolate, then it could also just be a background task. I would say it's expected that a task which uses the isolate is not executed anymore when the isolate is gone.
That being said, I don't know how much debug printing is added in node.js, so maybe it's okay to also print not so useful information.

Reply:
I mean, we’ve had enough issues with getting the platform implementation right that it might be worth adding optional debugging (in the style of `NODE_DEBUG_NATIVE` as we do for other parts of core) here. But that’s also out of the scope of this PR, I would say.

Reply:
If it's a-OK for tasks like this to go ignored, I guess my questions are answered (no, they won't be helpful). It would be nice to have `NODE_DEBUG_NATIVE` that does not require an Environment but yeah, probably not worth the hassle for this particular PR.